### ISSUE TITLE
Too large for Pro Mini

### ISSUE THREAD

**Issue by sbishop61** [kriswiner/MPU9250#167]
Kris,
 please excuse a newbie but I can't get your sketches to fit within a Pro Mini.

I am trying to use 9250 + BMP280 and second set up 9250 + MS5611. Happy to have serial output no screen at this stage.

Any clue as to what  I need to configure the sketch etc so to fit the Pro Mini?

Thanks
Stephen

**Comment by kriswiner** [comment]
Yes, 12 kB is pretty small. I have no solution if you have to use a Pro
Mini (which I love too) except for paring the sketch to the bare minimum
maybe, but I haven't used one for years since I discovered the Teensy
(option 1) and then started making my own development boards. I think you
would like Ladybug (option2) which has the same form factor as the Pro Mini
but is programmed via the USB using the Arduino IDE.

https://www.tindie.com/products/TleraCorp/ladybug-stm32l432-development-board/

Kris
